services/user.py

```diff
@@ -0,0 +1,54 @@
+from db.models import get_user_model, User
```

Comment:
According to the checklist, you should use `get_user_model()` instead of importing `User` directly. The import of `User` is unnecessary here and should be removed.